CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## [1.28.0] - 2021-06-25 - Muuido (무의도, 舞衣島)
```

Comment:
I am not sure should we update the date or not.

Reply:
I'll update